plugins/widget/plugin.js

```diff
@@ -4041,6 +4043,9 @@
  * * The widget element will be inserted into DOM.
  *
  * @property {Function} insert
+ * @param {Object} options Options object added in **4.11.0**.
+ * @param {CKEDITOR.editor} options.editor Editor where the widget is going to be inserted to.
+ * @param {Object} [options.commandData] Command data passed to the invoking command, if any.
```

Comment:
After docs built additional params are not visible in docs.

Reply:
@jacekbogdanski Interesting find, we have actually many of those:

https://github.com/ckeditor/ckeditor-dev/blob/8e2b356/plugins/embedbase/plugin.js#L618-L619
https://github.com/ckeditor/ckeditor-dev/blob/8e2b356/plugins/embedbase/plugin.js#L625-L626
https://github.com/ckeditor/ckeditor-dev/blob/8e2b356/plugins/uploadwidget/plugin.js#L300-L302
https://github.com/ckeditor/ckeditor-dev/blob/8e2b356/plugins/widget/plugin.js#L4227-L4229

Just to give a few. It was my presumption that it works correctly.

It might happen that this is a regression after switching to Umberto, I'll investigate this.

Reply:
@mlewand Jaceks finding got my attention, and after super short research I found that:
https://github.com/senchalabs/jsduck/wiki/@param
`Documents parameters for methods and events.`
So @param can be defined on method, but not on property.

Reply:
It was actually working before, this is upstream issue, I have created an issue for that cksource/umberto#567.